[error]Invalid Version: 1.2.6.1+ent

Hi,

the action just throws an error:

```
Run innovationnorway/setup-vault@v1
  with:
    version: ~1.5
  env:
    SOPS_VERSION: 3.6.0
    DEVSCRIPTS_VERSION: 2.20.4
##[error]Invalid Version: 1.2.6.1+ent
```

@marcofranssen 
1) HashiCorp recently started publishing enterprise versions here: https://releases.hashicorp.com/vault/
2) Some of these versions, like `1.2.6.1+ent`, is not a valid semantic version, and this is why you see the error:
```
##[error]Invalid Version: 1.2.6.1+ent
```
3) We (me and @roberthstrand) is working on a fix and expect to release it within next 4-8 hours.
About the latest version not being published to the Marketplace, sorry about that, we will fix it soon.
We will post update here within next few hours.

--- a/src/installer.ts
+++ b/src/installer.ts
@@ -43,7 +43,7 @@ async function queryLatestMatch(versionSpec: string): Promise<string> {
       const supportedBuild = version.builds.find(
         build => osPlat === build.os && osArch === build.arch
       )
-      if (supportedBuild) {
+      if (supportedBuild && semver.valid(supportedBuild.version)) {
         versions.push(supportedBuild.version)
       }
     }

